ChopChop/ViewModels/OnlineRecipes/OnlineRecipeViewModel.swift

```diff
@@ -43,6 +44,8 @@ class OnlineRecipeViewModel: ObservableObject {
             .sink { [weak self] recipe in
                 self?.recipe = recipe
 
+                self?.recipeServingText = "\(recipe.servings.removeZerosFromEnd()) \(recipe.servings == 1 ? "person" : "people")"
```

Comment:
actually this particular format is used in all recipes, maybe we can generalise it as view component somewhere